everviz/pages/configuration.py

```diff
@@ -0,0 +1,38 @@
+import os
+from everviz.log import get_logger
+
+
+logger = get_logger()
```

Comment:
I went for a custom logger because I wanted to have a log file in the everviz folder that is created now in the Everest output folder, and have everviz responsible for its log messages. 

If I understand correctly you are suggesting to just use the `everest` logger configured by Everest for logging. I am not sure how that would work and I need to experiment with it but if that works then the everviz log messages will be in together with the Everest log messages. Is that better?

Reply:
Not quite, would like to set up the logging as just:
```
ìmport logging
logger = logging.getLogger(__name__)
```
Then relegate the rest of the responsibility to `everest`, however if everest is using a custom logger that will not work as inteded as those handlers and the format will not be configured. Though I do argue that is a problem for `everest`, and as a plugin `everviz` shouldn´t set up it´s own logging.

Reply:
And unlike for example `spinningjenny`, `everviz` is in the same process as `everest`, and they can therefore have the same logging, with `everest` setting the log level, for example.

Reply:
I looked a bit more at the logging code, and if you replace the logger in this module with: ```
ìmport logging
logger = logging.getLogger(__name__)
```
We should be good to go, everything else seems to be set up sensibly. While the second statement isnt really needed the advantage of this is that we get where the logging is comming from.

Reply:
I am not sure logging will continue to work as I am expecting currently with just making your suggested change. 

How about I just remove logging altogether and we can just add it consistently throughout the project with a later PR?

Reply:
I think it will work as expected, but either is fine by me 👍